src/backend/api/routes.js

```diff
@@ -5,10 +5,17 @@ const knex = require("../database");
 const AccessError = require("./access-error");
 const handleError = require("./error-handler");
 
+const routesListToDBTables = {
```

Comment:
Nice reusability of CRUD functionality and also error handling. And well done

Reply:
Thanks! Nice to read good stuff before sleep:)